From f8f98ecbab0b71bb738834dfb2da3ba3bd1e8775 Mon Sep 17 00:00:00 2001
From: Fabien Dessenne <fabien.dessenne@foss.st.com>
Date: Mon, 14 Sep 2020 17:05:25 +0200
Subject: [PATCH 297/529] ARM: dts: stm32: Add hwlock for irqchip on stm32mp151

Define a hwspinlock to be used by the irq controller driver and the
syscon driver.

Reviewed-on: https://gerrit.st.com/c/mpu/oe/st/linux-stm32/+/282563
Signed-off-by: Fabien Dessenne <fabien.dessenne@foss.st.com>
Signed-off-by: Arnaud Pouliquen <arnaud.pouliquen@foss.st.com>
Change-Id: I9960fe758e506611df110f60145516aab0c2d3e2
---
 arch/arm/boot/dts/stm32mp151.dtsi | 1 +
 1 file changed, 1 insertion(+)

--- a/arch/arm/boot/dts/stm32mp151.dtsi
+++ b/arch/arm/boot/dts/stm32mp151.dtsi
@@ -1274,6 +1274,7 @@
 			#interrupt-cells = <2>;
 			#address-cells = <0>;
 			reg = <0x5000d000 0x400>;
+			hwlocks = <&hsem 1 1>;
 			wakeup-parent = <&pwr_irq>;
 
 			exti-interrupt-map {
